src/injected/components/details-dialog.tsx

```diff
@@ -162,12 +167,28 @@ export class DetailsDialog extends React.Component<DetailsDialogProps, DetailsDi
     }
 
     private renderFileIssueButton(issueData: CreateIssueDetailsTextData): JSX.Element {
-        return (
+        const oldExperienceButton: JSX.Element = (
             <FileIssueDetailsButton
                 deps={this.props.deps}
                 issueDetailsData={issueData}
                 issueTrackerPath={this.state.issueTrackerPath}
-                restoreFocus={false}
+                restoreFocus={true}
```

Comment:
>true [](start = 30, length = 4)

Is it intentional to change this as part of this PR?

Reply:
No, will fix.